maquetado/scss/styles.scss

```diff
@@ -4,19 +4,19 @@
 
 .book-container {
   display: flex;
```

Comment:
You can use `row` class instead and use this for other css properties. It's a good idea to re use css layout classes, because that way you can know how it is supposed to look by just checking the html tags and their classes